doc/source/conf.py

```diff
@@ -156,6 +156,9 @@ def _stop_fluent_container(gallery_conf, fname):
     "thumbnail_size": (350, 350),
     "reset_modules_order": "after",
     "reset_modules": (_stop_fluent_container),
+    "capture_repr": (),
+    "remove_config_comments": True,
+    "abort_on_example_error": True,
```

Comment:
Unrelated to the issue, but a command that a noticed among the options that could be useful: `abort_on_example_error`, to stop building the examples and fail when an error occurs, which I think is what we expected it to do by default. Let me know if the behavior of continuing despite failures is preferred and I will remove this.